.github/workflows/on_push.yml

```diff
@@ -105,3 +111,89 @@ jobs:
         --container api-container \
         -- /opt/app/bin/liquid_voting \
         eval "LiquidVoting.Release.migrate"
+
+  teardown1:
+    name: Teardown data before smoke test
+    needs: [deploy]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Login to gcloud
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_email: ${{ secrets.GKE_EMAIL }}
+        service_account_key: ${{ secrets.GKE_KEY }}
+
+    - name: Teardown smoke test data
+      env:
+        GKE_CLUSTER: ${{ secrets.GKE_CLUSTER }}
+        GKE_ZONE: ${{ secrets.GKE_ZONE }}
+        GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
+      run: |
+        gcloud container clusters get-credentials $GKE_CLUSTER --zone $GKE_ZONE --project $GKE_PROJECT
+        pod=$(kubectl get pods -l app=api  -o json | jq -r '.items[].metadata.name' | head -1)
+        kubectl wait --timeout=120s --for=condition=Ready pod/$pod
+        kubectl exec -ti pod/$pod \
+        --container api-container \
+        -- /opt/app/bin/liquid_voting \
+        eval "LiquidVoting.Release.smoke_test_teardown"
+
+  smoke_tests:
+    name: Smoke Tests
+    needs: [teardown1]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Run k6 votes smoke test
+        uses: k6io/action@v0.1
+        with:
+          filename: smoke_tests/votes_test.js
+
+      - name: Run k6 delegations smoke test
+        uses: k6io/action@v0.1
+        with:
+          filename: smoke_tests/delegations_test.js
+
+      - name: Run k6 participants smoke test
+        uses: k6io/action@v0.1
+        with:
+          filename: smoke_tests/participants_test.js
+
+      - name: Run k6 voting results smoke test
+        uses: k6io/action@v0.1
+        with:
+          filename: smoke_tests/voting_results_test.js
+
+  teardown2:
+    name: Teardown data after smoke test
+    needs: [smoke_tests]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Login to gcloud
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_email: ${{ secrets.GKE_EMAIL }}
+        service_account_key: ${{ secrets.GKE_KEY }}
+
+    - name: Teardown smoke test data
+      env:
+        GKE_CLUSTER: ${{ secrets.GKE_CLUSTER }}
+        GKE_ZONE: ${{ secrets.GKE_ZONE }}
+        GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
+      run: |
+        gcloud container clusters get-credentials $GKE_CLUSTER --zone $GKE_ZONE --project $GKE_PROJECT
+        pod=$(kubectl get pods -l app=api  -o json | jq -r '.items[].metadata.name' | head -1)
+        kubectl wait --timeout=120s --for=condition=Ready pod/$pod
+        kubectl exec -ti pod/$pod \
+        --container api-container \
+        -- /opt/app/bin/liquid_voting \
+        eval "LiquidVoting.Release.smoke_test_teardown"
+        
```

Comment:
need a newline here

Reply:
there is a newline there

Reply:
![image](https://user-images.githubusercontent.com/21290/88797534-c3054b80-d1a3-11ea-968e-ec6b8fc05242.png)
could be a default system line ending thing

Reply:
did you refresh page after my 'add neline at EOF' push?

Reply:
ok, so you saw the issue mid-review and fixed it - good. But I didn't get a notif, and a reviewer won't be refreshing the PR mid-way. So after requesting a review, it's good practice to avoid new changes until the review is submitted. If the commit mid-review is really needed, let the reviewer know :)